src/index.ts

```diff
@@ -131,6 +137,7 @@ export interface Options {
   githubToken?: string,
   webhookProxy?: string,
   port?: number
+
```

Comment:
this stray blank line seems superfluous

Reply:
You are right! I’d usually remove that line and update the PR, but because I moved on and have another PR based on this one, I think I take the easy way and just leave it. 

Thanks for your review!